content/post/the-blacker-the-box.md

```diff
@@ -0,0 +1,74 @@
+
+<!--more-->
+
+In this post I will not be getting into the weeds of complexity penalization algorithms or even how to weigh the [tech debt](https://ai.google/research/pubs/pub43146) associated with additional complexity. Instead, I want to take a step back and discuss how I think about "prediction" problems at a more macro level, and how I value accuracy and complexity for different types of problems.
+
+The thesis of this post is:
+
+**The faster the feedback on prediction accuracy, the blacker the box can be. The slower the feedback, the more your models should be explicit and formal.**
+
+In this post I'll talk through some examples of fast feedback problems and what makes them more amenable to to black-box prediction algorithms (provided you have the proper infrastructure) as well as slower feedback problems and how one might approach predictions in those situations.
+
+## Fast Feedback
+
+The machine learning community spends the bulk of its time working on and talking about fast feedback problems. Problems with fast feedback are defined by 1) the ability to quickly evaluate the correctness of a prediction[^1] and 2) the ability to play the game near infinite amounts of time[^2]. Some examples of fast feedback problems are:
+
+* Chess: it's easy to verify which player has one or lost. Feedback takes only as long as the length of the game.
+* Conversion for an Ad Placement: Feedback to Google or Facebook on whether you clicked a given advertisement, and whether you subsequently converted[^3] is nearly instantaneous.
+* Movie Recommendations: For a given list of potential movies to watch, Netflix gets near instantaneous feedback when you do or do not watch some of the content they've elevated for you.
+
+[Tech-debt considerations](https://ai.google/research/pubs/pub43146) aside, for fast-feedback problems the box can be impenetrably black *as long as you have confidence in your ability to accurately measure the prediction precision.* 
+
+If I am designing an application for optimizing landing-page content for my e-commerce site (i.e., choose the content that converts best), then I do not care if my data scientist has rigged up a prediction pipeline that involves passing a SVM prediction through [an octopus](https://en.wikipedia.org/wiki/Paul_the_Octopus) so long as that model out-performs every other model we've tested in our production environment.
+
+The key to this scenario is that I'm able to quickly and easily evaluate the performance of candidate prediction models and compare the current production model to new candidate models for evaluation. Because I have an *objective* measure of predictive success, I do not need any understanding of what the model is doing under-the-hood in order to make use of it.
```

Comment:
`I do not need any understanding of what the model is doing under-the-hood in order to make use of it.`

That's a pretty strong claim. Don't you need some assurance that the model will provide consistent and repeatable results? Maybe you don't know how it works per se, but you can probably do some macro level / post-hoc sanity checks right?

Reply:
Provocateur!

Yeah I think it's fine. I was reading with specific ears and this stood out to me. But after re-reading it as a whole I don't think you'll get many objections.